src/pages/AppList/AppListComponent.tsx

```diff
@@ -16,14 +16,14 @@ import { AlignRightStyle, ThinStyle } from '../../components/Filters/FilterStyle
 import { KialiAppState } from '../../store/Store';
 import { activeNamespacesSelector } from '../../store/Selectors';
 import { arrayEquals } from '../../utils/Common';
+import MetricsDuration from '../../components/MetricsOptions/MetricsDuration';
 
-interface AppListComponentState extends ListComponent.State<AppListItem> {
-  rateInterval: number;
-}
+interface AppListComponentState extends ListComponent.State<AppListItem> {}
 
 interface AppListComponentProps extends ListComponent.Props<AppListItem> {
   rateInterval: number;
   activeNamespaces: Namespace[];
+  onRateIntervalChanged: (rateInterval: number) => void;
```

Comment:
Can we not use the "RateInterval" terminology and consistently use "Duration" for durations.  Someone correct me if I'm wrong, that this is not a rate interval as used in prometheus queries but rather the duration/time range of the queried data.  Also, the unit should likely be DurationInSeconds, not number.